bot/seasons/valentines/myvalenstate.py

```diff
@@ -0,0 +1,80 @@
+import collections
+import json
+import logging
+from pathlib import Path
+from random import choice
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'valentines', 'valenstates.json'), 'r') as file:
+    STATES = json.load(file)
+    # STATES.sort()
```

Comment:
Don't need the comment

Reply:
Done

Reply:
I dont know who this CyrolX is but he did a great job.